test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Xamls/InfiniteScrollListTests.cs

```diff
@@ -17,6 +17,7 @@
 
 namespace NuGet.PackageManagement.UI.Test
 {
+    [Collection(MockedVS.Collection)]
```

Comment:
Do you need to set a GlobalService provider in the constructor?

Reply:
Good catch, I missed this in the instructions. Although, my tests ran without issues. Guessing it helps with multiple test instantiations.
sample:
```
public TestFrameworkTests(GlobalServiceProvider sp)
    {
        sp.Reset();
    }
```

Reply:
You're right. Resolving this for now.